From e7d008ff98df2049cce1c4d941b13b320da336c9 Mon Sep 17 00:00:00 2001
From: Matt Jolly <Matt.Jolly@footclan.ninja>
Date: Mon, 29 Jul 2024 23:42:04 +1000
Subject: [PATCH] no-op enterprise_companion.

For some reason this pulls in updater components which breaks
builds on OpenRC systems (or any non-systemd system, really).

It doesn't seem essential so we'll just not build it.

Bug: https://bugs.gentoo.org/936673
Signed-off-by: Matt Jolly <kangie@gentoo.org>
--- a/chrome/enterprise_companion/BUILD.gn
+++ b/chrome/enterprise_companion/BUILD.gn
@@ -60,16 +60,16 @@ static_library("client") {
   deps = [ "//mojo/public/cpp/platform" ]
 }
 
-if (!is_official_build) {
-  executable("enterprise_companion") {
-    sources = [ "main.cc" ]
-    deps = [ ":base" ]
-
-    if (is_win) {
-      configs += [ "//build/config/win:windowed" ]
-    }
-  }
-}
+#if (!is_official_build) {
+#  executable("enterprise_companion") {
+#    sources = [ "main.cc" ]
+#    deps = [ ":base" ]
+#
+#    if (is_win) {
+#      configs += [ "//build/config/win:windowed" ]
+#    }
+#  }
+#}
 
 group("all") {
   testonly = true
-- 
2.45.2

